### ISSUE TITLE
How to only padding the bottom when use the Conv2D ?

### ISSUE THREAD

**Issue by kenjewu** [pytorch/pytorch#4110]
if I only want to padding the bottom of every input when I use Conv2D. How to do it .
Please help me . Thanks,

**Comment by dhpollack** [comment]
use the [functional padding method](http://pytorch.org/docs/master/nn.html#torch.nn.functional.pad)

**Comment by kenjewu** [comment]
Thank you

**Comment by fmassa** [comment]
@kenjewu if this issue has been addressed for you, could you close it?
